soap/src/java/com/zimbra/soap/admin/message/GetAccountRequest.java

```diff
@@ -53,6 +53,17 @@
     @XmlAttribute(name=AdminConstants.A_APPLY_COS, required=false)
     private ZmBoolean applyCos = ZmBoolean.ONE /* true */;
 
+    /**
+     * @zm-api-field-tag effectiveQuota
+     * @zm-api-field-description Flag whether want to return effective value (minimum of zimbraMailQuota & zimbraMailDomainQuota)
+     * <table>
+     * <tr> <td> <b>1 (true)</b> </td> <td> zimbraMailQuota attribute will contain effective value.
+     * <tr> <td> <b>0 (false)[default]</b> </td> <td> zimbraMailQuota attribute will contain actual ldap value set </td> </tr>
+     * </table>
+     */
+    @XmlAttribute(name=AdminConstants.A_EFFECTIVE_QUOTA, required=false)
+    private ZmBoolean effectiveQuota = ZmBoolean.ZERO /* false */;
```

Comment:
Does `ZmBoolean.ZERO` need to be set?  I couldn't find any statement which sets `ZmBoolean.ZERO` in other files.

Reply:
Done

Reply:
I mean, just defining a variable might work.
```
private ZmBoolean effectiveQuota;
```

Reply:
Yes.. we can... but in this case effectiveQuota will gave null as default. Need to taken care of this.
Declared variable as you said